web/github/connect/index.go

```diff
@@ -15,9 +17,10 @@ var index string
 var tmpl = template.Must(template.New("index").Parse(index))
 
 func ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	random := kittehs.Must1(typeid.WithPrefix(""))
 	data := map[string]string{
 		"address": os.Getenv("WEBHOOK_ADDRESS"),
-		"path":    shortuuid.New(),
+		"path":    random.String(),
```

Comment:
is there any additional security consideration here? will random be enough? seed?

Reply:
Good question. Added comment.